clair/clair.go

```diff
@@ -57,6 +58,7 @@ func (imageAnalysis ImageAnalysis) ShortName(l v1.Layer) string {
 //Config configure Clair from configFile
 func Config() {
 	uri = fmtURI(viper.GetString("clair.uri"), viper.GetInt("clair.port")) + "/v1"
+	host = viper.GetString("clair.host")
```

Comment:
It's maybe just me but I find the name `host` a bit misleading in the config file. Maybe `headerHost` would be more self descriptive ? What do you think ?